main.js

```diff
@@ -0,0 +1,37 @@
+var leaderBoardArray = []
+
+function writeLeaderboard(array) {
+  for (var i = 0; i < array.length; i++) {
+    var leaderBoardObject = {
+      mmr: array[i].rankedSeasons[5][10].rankPoints,
+      name: array[i].displayName,
+    }
+    leaderBoardArray.push(leaderBoardObject)
+  }
+  leaderBoardArray.sort(function (a,b){
+    return parseFloat(b.mmr) - parseFloat(a.mmr)
+  })
+  leaderboardUl()
+}
+
+function leaderboardUl() {
+  for (var i = 0; i < leaderBoardArray.length; i++) {
+    var createUl = document.createElement('ul')
```

Comment:
Try to extract this DOM creation into a pure function. Then use a loop to call the function multiple times.